### PR TITLE
[sitemap] Combined conditions with AND

### PR DESCRIPTION
**Pull request by lolodomo**
Related to openhab/openhab-core#3819

Signed-off-by: Laurent Garnier <lg.hc@free.fr>

### CODE REVIEW

File: ui/sitemaps.md [hunk 1]
```diff
@@ -584,30 +584,40 @@ If the parameter is not provided, the default is to display the Item.
 Visibility syntax:
 
 ```java
-visibility=[item_name operator value, item_name operator value, ... ]
+visibility=[item_name operator value, item_name operator value AND item_name operator value, ... ]
 ```
 
+You set as many conditions as you want.
```
Comment (stefan-hoehn) [inline ui/sitemaps.md:590]: You _can_ set

File: ui/sitemaps.md [hunk 2]
```diff
@@ -584,30 +584,40 @@ If the parameter is not provided, the default is to display the Item.
...
+
+A condition can be a single comparison or a combination of several comparisons separated by the AND operator.
+You can use as many AND-separated comparisons as you want.
```
Comment (stefan-hoehn) [inline ui/sitemaps.md:593]: Maybe combine this sentence with line 590?

File: ui/sitemaps.md [hunk 3]
```diff
@@ -618,11 +628,18 @@ The icon may be tinted depending on the state as well.
...
 ```java
-labelcolor=[item_name operator value = "color", ... ]
-valuecolor=[item_name operator value = "color", ... ]
-iconcolor=[item_name operator value = "color", ... ]
+labelcolor=[item_name operator value = "color", item_name operator value AND item_name operator value = "color", ... ]
+valuecolor=[item_name operator value = "color", item_name operator value AND item_name operator value = "color", ... ]
+iconcolor=[item_name operator value = "color", item_name operator value AND item_name operator value = "color",... ]
 ```
 
+You set as many conditions as you want, along with a color for each condition.
```
Comment (stefan-hoehn) [inline ui/sitemaps.md:636]: can

File: ui/sitemaps.md [hunk 4]
```diff
@@ -618,11 +628,18 @@ The icon may be tinted depending on the state as well.
...
+
+A condition can be a single comparison or a combination of several comparisons separated by the AND operator.
+You can use as many AND-separated comparisons as you want.
```
Comment (stefan-hoehn) [inline ui/sitemaps.md:639]: same as above

**Review by stefan-hoehn — CHANGES_REQUESTED** [review]
Thanks for the docs, only some minor requests.

File: ui/sitemaps.md (hunk 1 above)
Comment (lolodomo) [reply ui/sitemaps.md:590]: Ok

File: ui/sitemaps.md (hunk 3 above)
Comment (lolodomo) [reply ui/sitemaps.md:636]: Ok

File: ui/sitemaps.md (hunk 2 above)
Comment (lolodomo) [reply ui/sitemaps.md]: I removed this sentence, I believe it is already clear with the previous sentence.

File: ui/sitemaps.md (hunk 4 above)
Comment (lolodomo) [reply ui/sitemaps.md]: I removed this sentence, I believe it is already clear with the previous sentence.

**Comment by stefan-hoehn** [comment]
Same problem here with synology.md as in #2154

**Pull request merged**